build.gradle

```diff
@@ -14,9 +15,10 @@ apply plugin: 'com.mparticle.kit'
 android {
     defaultConfig {
         minSdkVersion 14
+        resValue "string", "apptentive_distribution_version", project.version
     }
 }
 
 dependencies {
-    api 'com.apptentive:apptentive-android:4.0.2'
-}
+    compile 'com.apptentive:apptentive-android:5.1.0'
```

Comment:
we would like to keep our dependency declarations in line with Gradle 3.0, this should probably remain `api` or be `implementation`